src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+    private Random random;
+
+    public Lottery(ColorSupplier colorSupplier, Random random) {
+        this.random = random;
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
+        return ball;
```

Comment:
Let's make the value 100 constant, as the checklist requires)
And also we can combine it into one line